src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/definition/CommonCaseFields.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition;
+
+/**
+ * Names of the fields used by Bulk Scan in service CCD case definitions.
+ */
+public class CommonCaseFields {
```

Comment:
so is it just common case fields (which we have somewhere) or common fields specific for bulk scan?

Reply:
The fields that we have somewhere are exception record fields - we know them all. In service cases we're only aware of those fields that we need to set or read - those are common to all case definitions.

I'll rename to `CommonServiceCaseFields` if you're happy.

Reply:
Actually, this is one class to many for this PR - I'll remove it.

Reply:
done

Reply:
done - removed the class